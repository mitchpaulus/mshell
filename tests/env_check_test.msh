[($asdfasdfasdf?) ("Found") ("Environment variable asdfasdfasdf not found")] if wl
"My var value" $asdf!
[($asdf?) ("Environment variable asdf Found") ("Environment variable asdfasdfasdf not found")] if wl
