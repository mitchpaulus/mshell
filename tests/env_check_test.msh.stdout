Environment variable asdfasdfasdf not found
Environment variable asdf Found
